src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -137,7 +156,7 @@ public function setValue(&$objectOrArray, $propertyPath, $value)
         $overwrite = true;
 
         try {
-            if (PHP_VERSION_ID < 70000) {
+            if (PHP_VERSION_ID < 70000 && false === self::$previousErrorHandler) {
```

Comment:
this is a bug fix when a property accessor is callinged an other property accessor